web/assets/css/transmission-app.scss

```diff
@@ -534,7 +534,8 @@ $video-image: '../img/film.svg';
 
       &.compact {
         flex: 1;
-        font-size: x-small;
+        font-size: 0.875em;
+        font-weight: normal;
```

Comment:
Erase all in this selector leaving only flex: 1;

Reply:
Okay